docs/tutorials/tutorial_start_EOS-S3_HW.md

```diff
@@ -0,0 +1,140 @@
+# Getting started QuickLogic EOS S3 Board
+
+## Intro
+
+The main goal of this task was to get familiar with the board and development
+environment. QuickFeather EOS S3 Board is simple solution for getting started
+with FPGA development.
+SparkFun QuickLogic Thing Plus EOS S3 is development board, based on EOS S3
+FPGA/ARM Cortex-M4 Microcontroller.
+
+## Software development
+
+Software development for this board involves both programming ARM Cortex-M4
+and FPGA module.
+
+### Development of the FPGA module
+
+To develop the bitstream of the FPGA Synthesis the tool named _Symbiflow_
+is used.
+
+### Development of the ARM Cortex-M4
+
+To develop the M4 part of the SoC standard ARM toolchain
+could be used.
+
+### Basic steps
+
+To prepare development environment steps from [this instruction][build-instr-url]
+should be used(parts _Docker image_, _Building_).
+
+> Note: There was problem during compilation for Cortex-M4 device.
+
+```shell
+make -f makefiles/Makefile_Startup
+make[2]: Entering directory '/home/qorc-sdk/workspace/qorc-onion-apps/qorc_fpga_breathectrl/GCC_Project'
+make[2]: *** No rule to make target '/home/qorc-sdk/workspace/qorc-onion-apps/qorc_fpga_breathectrl/GCC_Project/output/startup_EOSS3B_GCC.o', needed by 'all'.  Stop.
+make[2]: Leaving directory '/home/qorc-sdk/workspace/qorc-onion-apps/qorc_fpga_breathectrl/GCC_Project'
+make[1]: *** [Makefile:59: Startup] Error 2
+make[1]: Leaving directory '/home/qorc-sdk/workspace/qorc-onion-apps/qorc_fpga_breathectrl/GCC_Project'
+make: *** [Makefile:54: m4] Error 2
+```
+
+There is workaround for this problem. During start of the Docker container
+one option should be corrected:
+
+```shell
+ -v $PWD:/home/qorc-sdk/workspace
+```
+
+to
+
+```shell
+ -v $PWD:/home/qorc-sdk/qorc-sdk/workspace
+```
+
+Warning: this correction reorganizes structure of the catalog in Docker container.
+
+More generic solution is to pay attention about nesting the directories.
+
+> Note: with newest environment, run by command:
+```shell
+$ docker run --rm -it -v $PWD:/home/qorc-sdk/workspace \
+    --device=/dev/ttyACM0:/dev/ttyS\_QORC ghcr.io/dasharo/twpm-sdk:main
+```
+After succesful download it stops during execution `make`, with error, there
+is no `yosys` utility. There is workaround for this problem, by using older
+version of the container.
+```shell
+$ docker run --rm -it -v $PWD:/home/qorc-sdk/workspace \
+    --device=/dev/ttyACM0:/dev/ttyS\_QORC 8f6e8a2450a3080d1f6d121214b88bc461299c201c61205d373f3510b2dfc3b4
```

Comment:
I'm getting error:

```
docker: Error response from daemon: No such image: sha256:8f6e8a2450a3080d1f6d121214b88bc461299c201c61205d373f3510b2dfc3b4.
```

This is probably an image you built locally, not a remote one.

Reply:
`ghcr.io/dasharo/twpm-sdk@sha256:ca99beadbc692e921762fdbe478d79bdfad3afd3db8d3e4084041bd51caaf6af` works, as Maciej mentioned on Matrix.